dialog/confirm.go

```diff
@@ -20,6 +20,13 @@ func (d *ConfirmDialog) SetConfirmText(label string) {
 	d.win.Refresh()
 }
 
+// SetConfirmImportance sets the importance level of the confirm button.
+//
+// Since 2.4
+func (d *ConfirmDialog) SetConfirmImportance(importance widget.ButtonImportance) {
```

Comment:
All other dialogs literally do it this way. I think it feels more inconsistent to it the other way, especially since you can't actually create dialogs by just creating the structs (all of the data is set up in the `NewXyz` functions).